### DISCUSSION TITLE
Example BLTouch Marlin start G Code loops probing sequence or crashes Ender 3v2 4.2.2

### DISCUSSION THREAD

**Discussion by Router-Jockey** [mriscoc/Ender3V2S1#303] [Q&A]
I tried the August maintenance build for Ender 3 v2 4.2.2 BL Touch and Unified. 
BLTouch version caused loops when sending;

G28           ; Home XYZ.
G29 P1        ; Do automated probing of the bed.
G29 P3        ; Smart Fill Repeat until all mesh points are filled in, Used to fill unreachable points.
G29 S0        ; Save UBL mesh points to slot 0 (EEPROM).
G29 F 10.0    ; Set Fade Height for correction at 10.0 mm.
G29 A         ; Activate the UBL System.

The August maintenance Unified Bed Leveling build worked better but X-Axis stopped working when printing. Probing went fine but when it went to run the priming strip down the side of the bed surface it stayed in the same X position and made a mess.

I restored defaults between each flash, no change. Went back to the previous maintenance release Ender3V2-422-BLTUBL-20220527.bin and everything works again. At least I know my printer isn't the reason. :) 

Some other things worth noting although trivial were the click-wheel sensitivity is VERY low on the August maintenance build which makes it take forever to reach 433 e-steps from the default 93 - need to set this for my Sprite Pro extruder.

Happy to do some more testing if it helps.

**Comment by Router-Jockey** [comment]
Edit. Sorry, I mean to say G29 S1, not S0. S0 caused a printer reboot... looks similar to a bug which was filed by someone else in the Marlin source. I've used S1 forever...

G28 ; Home XYZ.
G29 P1 ; Do automated probing of the bed.
G29 P3 ; Smart Fill Repeat until all mesh points are filled in, Used to fill unreachable points.
G29 S1 ; Save UBL mesh points to slot 0 (EEPROM).
G29 F 10.0 ; Set Fade Height for correction at 10.0 mm.
G29 A ; Activate the UBL System.

**Comment by Router-Jockey** [comment]
I tried this firmware version again and with the start G code you suggested it worked fine for a test print of an XYZ cube. Printed well enough other than slightly rounded corners on the Z axis top to bottom.
I lowered my temperature from 210 to 200 thinking I might be over-extruding slightly and tried again. The priming strip and skirt went fine but then when it started the actual print the extruder completely stopped turning. 
Since I had a 4.2.7 board I figured I would swap it and try the August firmware on that as well. 
Same thing. 
Checked for clogs, none. Preheated nozzle to 200 and manually extruded 10mm without issue.
I went back to the July release and it the issue is gone.

Suggestions??? I'm happy with the July version... just curious if this could be some kind of bug or not which will keep coming up in later versions if it goes unnoticed. I'm happy to help get to the bottom of this just let me know what I can do to help you identify it.

Thank you!

**Reply by mriscoc** [reply]
Did you check the Extrude Min Temperature? Maybe it is too high. If the temperature of the hotend is lower than this settings the extruder will not move.

**Comment by Router-Jockey** [comment]
I thought this too and I thought I knew where the setting was but I checked every menu for it multiple times and couldn't find it.

**Reply by mriscoc** [reply]
Advanced / Filament Settings

**Comment by Router-Jockey** [comment]
I saw that but it was 180. I thought that might be for the filament
changing menu function purposes.

**Reply by mriscoc** [reply]
No, it is a safety limit, no extruder movement is allowed if the hotend temperature falls below that value.
